server.js

```diff
@@ -1,12 +1,40 @@
-const path = require("path")
-const express = require("express");
+const path = require('path');
+const express = require('express');
+const config = require('./config');
+const token = config.token;
+const apiUrl = 'https://app-hrsei-api.herokuapp.com/api/fec2/hr-rpp';
```

Comment:
To follow the convention let's name all constant UPPERCASE. For example, APIURL. So we can identify it later quickly.

Reply:
Thanks Ash , 

In the above, you are  only referring to  CONFIG, TOKEN, and API , correct? 

I've never uppercased express, but am open if thats what you are suggesting.

Reply:
We don't need to uppercase config. Just TOEKN & APIURL